lib/Sema/TypeCheckPropertyWrapper.cpp

```diff
@@ -677,6 +678,10 @@ Expr *swift::buildPropertyWrapperInitialValueCall(
          ctx, typeExpr, startLoc, {initializer}, {argName},
          {initializer->getStartLoc()}, endLoc,
          nullptr, /*implicit=*/true);
+
+      if (!lastWrapper->getInitialValueInit()) {
+        lastWrapper->setInitialValueInit(initializer);
```

Comment:
The `initializer` is effectively going to get rewritten by `CSApply.cpp`, so this won't necessarily work.